docs/src/00-doc/02_modify_component.stories.mdx

```diff
@@ -2,7 +2,41 @@ import { Meta } from '@storybook/addon-docs/blocks';
 
 <Meta title="Documentation/How to modify existing components" />
 
-
 # How to modify existing components
 
-...
+Our design system is a living organism that all Wikidata/Wikibase team members should contribute to grow and evolve. In some cases, that contribution will take place by the direct supply of a newly implemented component to the library. In other occasions, it will take the form of an improvement or a correction applied to an existing system component. 
+
+Changes can be applied to existing library components whenever they don't perform as expected, or in case they don't reflect existing or new visual styles or interaction patterns documented by the Style guide and the component design specs.
+
+Depending on the type of modification needed, changes can be applied by modifying the system's ***design tokens*** – in case the visual style of components need to be corrected or updated – or by directly adjusting the ***component's code***. 
```

Comment:
```suggestion
Depending on the type of modification needed, changes can be applied by adjusting the values of system's ***design tokens*** – in case the visual style of components need to be corrected or updated – or by directly modifying the ***component's code***. 
```